project/Build.scala

```diff
@@ -81,8 +81,8 @@ object Build {
    *  set to 3.1.3. If it is going to be 3.1.0, it must be set to the latest
    *  3.0.x release.
    */
-  val previousDottyVersion = "3.0.0-RC3"
-  val previousDottyBinaryVersion = "3.0.0-RC3"
+  val previousDottyVersion = "3.0.0"
+  val previousDottyBinaryVersion = "3.0.0"
```

Comment:
The binary version is 3 now (can just be hardcoded in the single place where it's used)